collection.php

```diff
@@ -3,24 +3,21 @@
 <head>
     <title>Collection</title>
     <link rel="stylesheet" type="text/css" href="collection.css">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
```

Comment:
Link to your normalize.css file should always be listed before any of your own stylesheets. So switch the order of these link tags!